src/array.ts

```diff
@@ -1,31 +1,27 @@
-export const objectListForAttribute = <T>(
+export const groupByKey = <T>(
```

Comment:
I think what you are doing is grouping  by key so I wanted to get a more concise name for it as opposed to the more abstract `objectListForAttribute`.

Reply:
makes sense, I always way overthink naming and it usually degrades after the first few minutes of thought